File: CodigoDelitos/delitos/direccion.java
package delitos;

public class direccion {
	private String Departamento;
	private String Municipio;
	private String AldeaCiudad;
	private String Colonia;
	private String Codigopostal;
	private String Bloque;
	private String Casa;
	
	public direccion(String departamento, String municipio, String aldeaCiudad, String colonia, String codigopostal,
			String bloque ) {
		super();
		Departamento = departamento;
		Municipio = municipio;
		AldeaCiudad = aldeaCiudad;
		Colonia = colonia;
		Codigopostal = codigopostal;
		Bloque = bloque;
		
	}

	public String getDepartamento() {
		return Departamento;
	}
	public void setDepartamento(String departamento) {
		Departamento = departamento;
	}

	public String getMunicipio() {
		return Municipio;
	}
	public void setMunicipio(String municipio) {
		Municipio = municipio;
	}

	public String getAldeaCiudad() {
		return AldeaCiudad;
	}
	public void setAldeaCiudad(String aldeaCiudad) {
		AldeaCiudad = aldeaCiudad;
	}

	public String getColonia() {
		return Colonia;
	}
	public void setColonia(String colonia) {
		Colonia = colonia;
	}

	public String getCodigopostal() {
		return Codigopostal;
	}
	public void setCodigopostal(String codigopostal) {
		Codigopostal = codigopostal;
	}

	public String getBloque() {
		return Bloque;
	}
	public void setBloque(String bloque) {
		Bloque = bloque;
	}

	public String getCasa() {
		return Casa;
	}
	public void setCasa(String casa) {
		Casa = casa;
	}

	@Override
	public String toString() {
		return "direccion [Departamento=" + Departamento + ", Municipio=" + Municipio + ", AldeaCiudad=" + AldeaCiudad
				+ ", Colonia=" + Colonia + ", Codigopostal=" + Codigopostal + ", Bloque=" + Bloque + ", Casa=" + Casa
				+ "]";
	}
	
	public void mostrardireccion() {
	    System.out.println("Departamento: " + Departamento);
	    System.out.println("Municipio: " + Municipio);
	    System.out.println("AldeaCiudad: " + AldeaCiudad);
	    System.out.println("Colonia: " + Colonia);
	    System.out.println("Codigopostal: " + Codigopostal);
	}
	
	

}

File: CodigoDelito/VersionWEB/php/Direccion.php
<?php


class Direccion
{
    private $departamento;
    private $municipio;
    private $colonia;
    private $numeroBloque;
    private $numeroCasa;
    private $codigoPostal;
   
    //contructor 
    //a diferencia de java php no permite tener mas de un constructor por loq ue fue necesario indicar
    //que algunos paramentros serian opcionales, ya que delito requiere de una direccion, pero no
    //no requiere los mismo paramentro que la direccion de las personas
    public function __construct($departamento, $municipio, $colonia, $numeroBloque, $numeroCasa = null, $codigoPostal= null)
    {
        $this->departamento = $departamento;
        $this->municipio = $municipio;
        $this->colonia = $colonia;
        $this->numeroBloque = $numeroBloque;
        $this->numeroCasa = $numeroCasa ?: 'no requiere numerocasa'; 
        $this->codigoPostal = $codigoPostal ?: 'no requiere codigo postal';
    }

    


    //getters
    public function getDepartamento(){
        return $this->departamento;
    }
    public function getMunicipio(){
        return $this->municipio;
    }
    public function getColonia(){
        return $this->colonia;
    }
    public function getNumeroBloque(){
        return $this->numeroBloque;
    }
    public function getNumeroCasa(){
        return $this->numeroCasa;
    }
    public function getCodigoPostal(){
        return $this->codigoPostal;
    }




    //setters
    public function setMunicipio($municipio){
        $this->municipio = $municipio;

        return $this;
    }
    public function setDepartamento($departamento){
        $this->departamento = $departamento;

        return $this;
    }
    public function setColonia($colonia){

        $this->colonia = $colonia;

        return $this;
    }
    public function setNumeroBloque($numeroBloque){
        $this->numeroBloque = $numeroBloque;

        return $this;
    }
    public function setNumeroCasa($numeroCasa) {
        $this->numeroCasa = $numeroCasa;

        return $this;
    }
    public function setCodigoPostal($codigoPostal){
        $this->codigoPostal = $codigoPostal;

        return $this;
    }

    public function toString(){
        return "Direccion: ". $this->getDepartamento(). ", ".$this->getMunicipio().", colonia " .$this->getColonia(). ", Bloque ". $this->getNumeroBloque() . ", casa ". $this->getNumeroCasa();
    }
}








?>